pkgs/applications/misc/prusa-slicer/default.nix

```diff
@@ -47,14 +47,6 @@ let
     pname = "wxwidgets-prusa3d-patched";
     version = "3.2.0";
     configureFlags = old.configureFlags ++ [ "--disable-glcanvasegl" ];
```

Comment:
Is this still required? Otherwise we wouldn't need the override at all.

Reply:
the official release notes have some notes about egl support being problematic, so i kept it. But the call is correct, i should try whether it works without.

Reply:
Presumably Fedora and co aren't building it that way.